numpy/core/src/umath/matmul.c.src

```diff
@@ -254,16 +256,18 @@ NPY_NO_EXPORT void
     npy_intp is1_m=steps[0], is1_n=steps[1], is2_n=steps[2], is2_p=steps[3],
          os_m=steps[4], os_p=steps[5];
     npy_intp ib1_n, ib2_n, ib2_p, ob_p;
-#if @USEBLAS@ & defined(HAVE_CBLAS)
+#if @USEBLAS@ && defined(HAVE_CBLAS)
+    int sz = sizeof(@typ@);
```

Comment:
This should be an `npy_intp`, since you compare it to one below. This line probably also warns on MSVC

Reply:
ok